### PR TITLE
small fixes: (doc 08-12)

### PR DESCRIPTION
**Pull request by tahmeedshah**

### CODE REVIEW

File: app/components/common/share-record.vue [hunk 1]
```diff
@@ -51,7 +51,7 @@
                     <span v-if="sharedData.type=='email'">
                       {{$t('emailSent')}}
                       <ul v-if="sharedData[sharedData.type].emailsSentTo && sharedData[sharedData.type].emailsSentTo.length>0">
-                        <li v-for="(email, index) in sharedData[sharedData.type].emailsSentTo" :key="index">{{email}}</li>
```
Comment (blaisefonseca) [inline app/components/common/share-record.vue:54]: undo this change, i have fixed it

File: app/css/template.css [hunk 2]
```diff
@@ -3412,4 +3412,11 @@ embed-footer{
 }
 .grecaptcha-badge:hover {
     width: 256px !important;
+}
+
+.km-rtl trix-editor{
+    width: calc(100% - 44px);
+}
+.km-rtl .input-group-text{
+    width: 46px;
```
Comment (blaisefonseca) [inline app/css/template.css:3421]: will it be responsive?

Comment (tahmeedshah) [reply app/css/template.css:3420]: yes it will be

**Review by blaisefonseca — APPROVED** [review]
